mantle.gemspec

```diff
@@ -6,8 +6,8 @@ require 'mantle/version'
 Gem::Specification.new do |gem|
   gem.name          = "mantle"
   gem.version       = Mantle::VERSION
-  gem.authors       = ["Grant Ammons", "Brandon Hilkert", "Scott Gibson", "Frank Hmeidan"]
-  gem.email         = ["gammons@gmail.com", "brandonhilkert@gmail.com", "sevgibson@gmail.com", "frank.hmeidan@gmail.com"]
+  gem.authors       = ["Scott Gibson", "Frank Hmeidan"]
+  gem.email         = ["sevgibson@gmail.com", "frank.hmeidan@gmail.com"]
```

Comment:
👋

Reply:
:wave: !

Reply:
@brandonhilkert hahaha there you are! Want me to leave you guys on? I thought this was taken private but 🤷‍♂️

Reply:
hey @gammons ! Hope you're both well.

Reply:
Doing great!   Hope everyone in PLD land is doing ok as well!

Reply:
👋 😄